cli/azd/cmd/config.go

```diff
@@ -65,6 +63,9 @@ func configListCmdDesign(global *internal.GlobalCommandOptions) (*cobra.Command,
 		Use:   "list",
 		Short: "Lists all configuration values",
 		Long:  `Lists all configuration values in ` + userConfigPath + `.`,
+		Annotations: map[string]string{
+			commands.RequireNoLoginAnnotation: "true",
+		},
```

Comment:
What is the annotation used for? Is it temporarily for the case where commands are still using `azcli` via context? Should we remove this as part of #899?

Reply:
Yes - Hopefully the [doc comment on the symbol](https://github.com/Azure/azure-dev/pull/1022/files#diff-906ea9144e270078a7ec004805cbdea5050bafa628e2eaf97206bd6b76b80cb6R122) (which is in builder.go much later in this PR) explained what was going on here.  I will add a `TODO(azure/azure-dev#899)` reference to that comment so we know to go clean it up when we solve that issue.